src/chocolatey.resources/helpers/functions/Uninstall-ChocolateyPath.ps1

```diff
@@ -0,0 +1,174 @@
+# Copyright 2011 - Present RealDimensions Software, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+function Uninstall-ChocolateyPath {
+<#
+.SYNOPSIS
+**NOTE:** Administrative Access Required when `-PathType 'Machine'.`
+
+This puts a directory to the PATH environment variable of the
+requested scope (Machine or User).
+
+.DESCRIPTION
+Removes path from target path scope.  Removes multiple occurances (if they exist)
+and all occurances with or without a trailing slash.
+
+.NOTES
+This command will assert UAC/Admin privileges on the machine if
+`-PathType 'Machine'`.
+
+This is used when the application/tool is not being linked by Chocolatey
+(not in the lib folder).
```

Comment:
I will also need to update this to indicate first supported version of Chocolatey this is in.